dapps/digital-twin-detail/dbcp.json

```diff
@@ -0,0 +1,34 @@
me//eFlwCo8/f1ydCv8jHTzmC2AAjZfIy1QYDO4VoSAG3zMt7UVKiAhwCKS87As9tWKHr41P6vmpprymiAhwCIIObjT0kA5PtvmeU3PASw/su1sPnAJsXnL8aRwBgcERgNLwEQAczHR0gAp8BCGz7wEMDWg/nw/p53FZ9PNcCN3Yz3keUpAJmm5uMiEoByywsHYvA/KImQbQL+SIpLgt4pfWDaTRnQKbmLpjgqrlVA7vaVfg1AvqT1uwMe/dViU+6VuwAYCe0TIIGZjy0nAAJ7p25/gNLA/gBx0XGyP8DQ7sM1xXHy4gl4YuNjIc/Lyykw7T5FCYCIxSGiIyXRGgIgm36v5OvlozckgGo/Ivjx3BGoddZCQkwivDH7XUiMTVQdZ3VtFWS+N0u+TiDI3+CVmepnKpUiUgAEGYssIYAlw5erWgZ++koRLNm4UM68R9MWQVp/bW6Mm7/ZCOu/+pvf70iMT9+1SnMNowTRAiCECyApJhn+NPY11b9bnr9EbsP1jtOz3/8NXKq41OTzsX3S4PEJS0y9d1sADFoK3iOpp+LzL5VfhEc+yJJrgJzxC2HCDdr3EQiUCWn9JmDnb5Gp920LgBEVEQ0DWg8CV6kEEMIfoKKmHA6d/g4qayp09QEIakqe2bJUdizxhQxAyyavgNYJbUy7b1sAPqjxCaA2+slJT8OonreojodGAP/Y90FIU3B0ZDRMGngXpA+aonnIGQyrCMBShqCImkiAK5I8FPQHOY727dgfMoZNl3vpaqCagyyA5FGkKk0otnF902QHFCOFYAEBnG0RpuADP++XM17J7F8oqMahjmfP9r10X0u4ACQ4HtaTQZTh737+NhReOmF4OkkI5I6mthbyRrwApD1hOx28r/BreGHHc6amlayRZCjS6pQqWgBShPT3sHUI2bD3A7mjZzbPTXsBBnYZrOm3FhDAH8PWJcwWgAIkmGs5p9BhKSMho+eMgN8rdQq1BaAASbrNUm7hHsrOlQedCVTiFm4LQBHJllkY4k29sx6uFF4NuUoo2MIQWwAh+Sl/wfZUyywNG9Hp5kaf1VbUQm21/z2OaSnXkbPfB10aRlY+JU6feiG3ca17FIgUgCRJeXk5BdPtxaECEVwDLMEa4M/28nCBCBWAJI3KzynYa28QIRCBAijF0i9XuZbZIoYmfy6fvAIup/LJyZa4RYwReNp/+d+eD61gEKq6WgUVFyoVnUuZv3TyM9DZhGlaXgisAeZgDbCW/mGpbeKoFoi5FAf9UwYEPKelbhNnIE7M9S75OdvliC23UeSodqPht0Pni0wCN4QIQJL+g52/cQ1/en9nbxXLF0E1wHys/t/0/GFvFi0QAQIg+3oKCqDBCdLeLl4gvAUgRUrr8+YVNLKd2y+MEIiAxaFjsf1v9BJQ+5UxAuEqAAkOYc9/cNOP/WC/NMp8Gr00ig8NY39vgr02jqaIU3k/mJb22jhOFEkRUs+8+QXKXhtH2C+ONBfaoo62quMBZv6TmPkv+/0u0I/sV8eaB+1QSu1/sKXpBlIoSdKAvJyCKn9f2i+PFoDSHcqMAEt/Fpb+dwJ+H+zH9uvjjWfT/g3w4dec+tfY88fSP9Rf2//LKSFAEYzBw+5Q55kBdQizBi2AwR2EjEgNh5efYgMSpOHQ79PgpygARUCSnc0v5Y25vW0aZAy8X/MycNGcLTkjd/qUbE1rFFj1b8CSH9K/Q6kAaKqYhoVCPIZomrj+sgtu7XY7jOgxCrq27mbIhtFmUlxSDIUXT8CeE1/AnuOfQ72rnmf0pZizfTxTvsFQvLm8yA4hQSKouFAB1SXX9F8szMHSn42lX9GsrqpXTKAINuPhHpE3V3GpEqouV+m/UJiCmV+AmT9V8flqLs6aAponELqhRNWVKqi4qMx1rIVRjDk6REnV70H1GwhRBOQxtAMEv2GUXipZdrZc/4XCByeW/rux9O9U8yNNr6BEESzDw4ui77imogZKi8tVvWcobJGklfk5Bc+r/pnW+ES7kXugjaVKz5QpettYuIIlPw9L/nRNv9UaKXMfo+1lRoh+AHU1TigpKlG1piBskLBP5oLx3m5e6n6uAxRBJ3B7D6WKfg7OunooLSoFZ61TdFJ4cgpzcDR2+k5rvYDu11CjCMhYTzVBJ9FPo55EUFwGddV1+i9mfch5Nw1L/mE9F9EtAIKJgCaNHKKfCvUFSs6UQl1VWIuAqvtb9WY+YYgACDZpRMND8SLAUUEZjg5olBCGVEZERdy1KXvbZ0ZczDABEFZqDtym40qoLqkWnRQjMaTa98ZQARBMBFQTpPJ7LoEJI9MxbembbmTmE4YLgGCjg21ggSEi0exNx5Jsfp+qp7cf+NImwewENCMl3FhENFfTMRl5sGM7V+s4P+T1zb4BZjbOBcFzB0QzMx07QZKe1WLeVYPpAiDYBBL5Egjflr6ZmI6LseTPVTuxowUuAiDYVDI1CUL9CQgrm44x47fiCCZbzZSurvh43yDzLHoFBLmXebCg6bgUM/8JpZ48RsFdAASrDUgEwhxNCauYjsmBE0v9Y7xKfaO4Rd44sx6+DoJWIhNCTccSHML/Lwzlum1uEgTDFp9QTbAKBBmPBJiOC2mvfjyuDbZogwfCBeCBrUWch4Fe1pfKO37ZdHyxEqqvmmo6poxfAy5YE2itHm8sIwAPrEYg4xG9tpN702CK6ZiqehesxsxfL7rEN02ahWHb1VCtQILgtnGVQabja1KktNFVD2/5bstiJSwtAA/MrEwimAnuN5yZblXUaDom690X2J7QAsAPzTLfGkmzEIA3bAg5GcMUdjTNnlBT6bYahjAdl0qStAv7EAX4NP8lYiinh2YnAF/YVve08yU1FzT7aOibT2TTcXGZt9XwJ8zwfZjh/8XSvpu2XBf9DPTQ7AXgC2su+rPQD9zzD6ngdlJxQOj5iCIMVPefxVDorHUWXSur+RGbhOOuetehTdnbLF+tq+H/L3kPS+Qz1c4AAAAASUVORK5CYII=",
```

Comment:
where is the Base64 string comes from? Seems to be invalid base64

Reply:
It's copypaste from a different dapp tbh